aws/resource_aws_waf_rule_test.go

```diff
@@ -402,15 +402,15 @@ func testAccAWSWafRuleConfig_changePredicates(name string) string {
 	return fmt.Sprintf(`
 resource "aws_waf_ipset" "ipset" {
   name = "%s"
-  ip_set_descriptors {
```

Comment:
Was this change intentional?

Reply:
Sorry, this is my mistake. Too mach changes were in the commit.